formlibrary/forms.py

```diff
@@ -75,18 +76,18 @@ def __init__(self, *args, **kwargs):
 
         super(DistributionForm, self).__init__(*args, **kwargs)
 
-        self.fields['initiation'].queryset = ProjectAgreement.objects.filter(
-            program__organization=self.request.user.activity_user.organization)
+        # self.fields['initiation'].queryset = ProjectAgreement.objects.filter(
```

Comment:
Just curious, is there a reason why these fields are commented out and not deleted since they're unused?

Reply:
@Allan690 I believe this is because refactoring is still in progress so a lot things are still commented out. cc: @TAnas0